style.css

```diff
@@ -0,0 +1,28 @@
+html,
+body {
+   margin:0;
+   padding:0;
+   height:100%;
+}
+#container {
+   min-height:100%;
+   position:relative;
+}
+#header {
+   background:black;
+   padding:1px;
+   color: white;
+}
+#body {
```

Comment:
avoid using built-in tags, instead use custom classes as these may cause side effects in the future